lib/ancestry/materialized_path.rb

```diff
@@ -97,6 +97,16 @@ def sane_ancestry?
         (ancestry_value.nil? || !ancestor_ids.include?(self.id)) && valid?
       end
 
+      def parent= parent
```

Comment:
think this will take care of itself if you change `ancestor_ids=`

Reply:
but not sure if you want to implement `ancestry=`
that seems a good place to mess up our cache